README.md

```diff
@@ -3,55 +3,52 @@
 [![Build Status](https://api.travis-ci.com/GrayJack/coreutils.svg?branch=master)](https://travis-ci.com/GrayJack/coreutils)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils?ref=badge_shield)
+<!-- [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils?ref=badge_large) -->
 
 
-An attempt to make a safe re-implementation coreutils aiming for a minimal and yet complete set of utilities. This project aims to have **at least** POSIX especification and the common features between several implementations of the utility.
-
-This project has no intent to be 100% compatible with GNU's coreutils, like [Uutils' coreutils](https://github.com/uutils/coreutils), but if happens to be, that's ok too.
+An in-progress safe implementation of `coreutils` aiming for a minimal and yet complete set of utilities. This project aims at the _POSIX_ specification basic requirements, plus common and useful features present in other implementations of the utility.
 
-## Contributing
-Check the [CONTRIBUTING.md](./CONTRIBUTING.md) file for the guidelines to contribute to the project, including issue reports, git commits messages, etc.
+This project has no intent to be compatible with _GNU's coreutils_, like [Uutils' coreutils](https://github.com/uutils/coreutils).
 
 ## Minimum Rust Version Policy
-This project's minimum supported `rustc` version (MSRV) is 1.40.0.
+This project's minimum supported `rustc` version (_MSRV_) is _1.40.0_.
 
-This will not be conservative until we get to a 1.0 version. So it can be changed at any point in time.
+This will not be conservative until we get to a _1.0_ version. So it can be changed at any point in time.
 
 <!-- In general, this project will try to be conservative with respect to the minimum supported version of Rust, but in case of safety reasons it may bump at any time [e.g. `MaybeUninit` stabilization on 1.36.0 fixing huge problems with `std::mem::uninitialized()`] or improvements that affect positively conditional compilation (we use it a lot). -->
 
 ## Compilation tests guarantees
-The compilations is tested for Rust Tier 1 and Tier 2 x86_64 Unix/Unix-like platforms, with exception of redox target, with Github Actions.
+Compilation is tested for Rust Tier 1 and Tier 2 _x86_64 Unix/Unix-like_ platforms (except _Redox_), with _CI_.
 
-The Tier1 platforms are tested on stable, beta and nightly Rust, while Tier2 and Tier3 platforms are tested on stable only, since they are no guarantees to be available in beta and nightly Rust.
+All platforms are tested on _stable_ _Rust_, and Tier 1 platforms are also tested on _beta_ and _nightly_ _Rust_ (Tier 2 and 3 only guarantee _stable_ full capacity).
```

Comment:
All platforms are tested on MSRV and stable Rust

Reply:
ok